fiobank.py

```diff
@@ -104,7 +115,7 @@ def _parse_info(self, data):
     def _parse_transactions(self, data):
         schema = self.transaction_schema
         try:
-            entries = data['accountStatement']['transactionList']['transaction']
+            entries = data['accountStatement']['transactionList']['transaction']  # noqa
```

Comment:
OK, approved. Splitting this into two lines just because of one char would be pretty much dumb :-) :+1: